bot/exts/info/information.py

```diff
@@ -537,17 +549,44 @@ async def rules(self, ctx: Context, rules: Greedy[int]) -> None:
 
         # Remove duplicates and sort the rule indices
         rules = sorted(set(rules))
```

Comment:
Do you think renaming this to "indices", "numbers", or something similar would make the distinction between this and the keywords more clear?